RHDEVDOCS-5513: Uninstalling OpenShift Builds (#63619)
openshift/openshift-docs: builds/uninstalling-openshift-builds.adoc

:_content-type: ASSEMBLY
[id="uninstalling-openshift-builds"]
= Uninstalling {builds-shortname}
include::_attributes/common-attributes.adoc[]
:context: uninstalling-openshift-builds

toc::[]

Cluster administrators can uninstall the {builds-title} Operator by performing the following tasks:

. Delete the Custom Resources (CRs) that were added by default when you installed the {builds-title} Operator.

> . Uninstall the {pipelines-title} Operator.

Reviewer: ```suggestion
. Uninstall the {builds-title} Operator.
```

:_content-type: ASSEMBLY
[id="uninstalling-openshift-builds"]
= Uninstalling {builds-shortname}
include::_attributes/common-attributes.adoc[]
:context: uninstalling-openshift-builds

toc::[]

> Cluster administrators can uninstall the {builds-title} Operator by performing the following tasks:

Reviewer 2: ```suggestion
Cluster administrators can uninstall the {builds-title} Operator.

Perform the following tasks to uninstall the {builds-title} Operator:
```

Reviewer 3: Use second person.
```suggestion
 As an administrator, you can uninstall the {builds-title} Operator by performing the following tasks:
```

> * Delete the Custom Resources (CRs) that were added by default when you installed the {builds-title} Operator.

Reviewer 2: ```suggestion
* Delete the custom resources (CRs) that were added by default when you installed the {builds-title} Operator.
```

> * Uninstall the {builds-title} Operator.
> 
> Uninstalling only the Operator will not remove the {builds-title} components created by default when the Operator is installed.

Reviewer 2: ```suggestion
* Uninstall the {builds-title} Operator.
+
NOTE: Uninstalling only the Operator does not remove the {builds-title} components that were created by default when you installed the Operator.
```

Reviewer 3: I would suggest reverse the sequence of tasks (unless there's a technical mandate to follow this sequence which I believe there isn't) which would be:

1. Uninstall Operator
2. Delete CRs. 
    Note: Uninstalling the Operator does not automatically remove the CRs and the related components.

Author: @snarayan-redhat I got it confirmed from SME and we cannot reverse the sequence. I have changed the bullet points to numbered sequence because the task of deleting CRs must be done before uninstalling the Operator.

include::modules/ob-deleting-the-builds-component-and-custom-resources.adoc[leveloffset=+1]

include::modules/ob-uninstalling-the-builds-operator.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

> * You can learn more about uninstalling Operators on {product-title} in the link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-deleting-operators-from-cluster.html#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster] section.

Reviewer 2: ```suggestion
* link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-deleting-operators-from-cluster.html#olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster]
```

Reviewer 2: xref link?

Author: same justification given in the first comment

:_content-type: ASSEMBLY
[id="uninstalling-openshift-builds"]
= Uninstalling {builds-shortname}
include::_attributes/common-attributes.adoc[]
:context: uninstalling-openshift-builds

toc::[]

As a cluster administrator, you can uninstall the {builds-title} Operator. Uninstalling the Operator does not automatically remove the custom resources (CRs) and {builds-shortname} components. 

> To uninstall, perform the following tasks:

Reviewer 4: Nit/suggestion: "To uninstall the {build-shortname} Operator, perform the following tasks:"

. Delete the custom resources (CRs) that were created by default when you installed the Operator.
+

> [CAUTION]

Reviewer 4: Caution is not a supported admonition type. See the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions).
